gobblin-utility/src/main/java/gobblin/util/SerializationUtils.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.util;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.nio.charset.Charset;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Closer;
+
+
+/**
+ * A utility class for serializing and deserializing Objects to/from Strings.
+ *
+ * @author ziliu
+ */
+public class SerializationUtils {
+
+  private static final Charset CHARSET = Charsets.ISO_8859_1;
+
+  /**
+   * Serialize an object into a String. The object is first serialized into a byte array,
+   * which is converted into a String using charset ISO_8859_1.
+   *
+   * @param obj A {@link Serializable} object
+   * @return A String representing the input object
+   */
+  public static <T extends Serializable> String serialize(T obj) throws IOException {
+    Closer closer = Closer.create();
+    try {
+      ByteArrayOutputStream bos = closer.register(new ByteArrayOutputStream());
+      ObjectOutputStream oos = closer.register(new ObjectOutputStream(bos));
+      oos.writeObject(obj);
+      oos.flush();
+      return bos.toString(CHARSET.name());
```

Comment:
I suggest you add an additional parameter of type `BaseEncoding` to this method and use that to encode to and decode from strings.